oct/ansible/oct/playbooks/package/ami-mark-ready.yml

```diff
@@ -55,4 +55,13 @@
         resource: '{{ origin_ci_aws_ami_id }}'
         tags:
           ready: 'yes'
-        state: present
\ No newline at end of file
+        state: present
+
+    - name: add any additional tags
```

Comment:
I'd add validation in the Python code that disallows this behavior on `--mark-ready`.